common/utils/loadUserData.js

```diff
@@ -173,6 +174,33 @@ async function syncCoworkerData(coworkerPayload, store) {
   await Promise.all(syncActions);
 }
 
+export async function syncMissions(missions, store, syncMethod) {
```

Comment:
I extracted the code that add missions to the store to be able to use it on history (when filter change, we retrieve missions and put them to the store with this function)

`syncMethod` param is used to be able to choose the action used when adding it to the store : 
- on login (loadUserData), we erase any missions and put the new ones (last 2 months)
- on history filter change, we add the new missions to the existing ones